### PR TITLE
Added minor changes

### PR DESCRIPTION
**Pull request by MohammadHasham**
1. destructured variables in fetchUser (ES6).
2. added catch in case the promise fails.

### CODE REVIEW

**Comment by endormi** [comment]
Thank you for this PR @MohammadHasham, but I'm not actually looking for contributors for this project. I do have other open source projects that I need help with for example https://github.com/devisle/advanced-react-cli and https://github.com/endormi/dj

File: js/app.js [hunk 1]
```diff
@@ -24,13 +24,14 @@ const fetchUser = async (user) => {
...
-        reposContainer.innerHTML = `Repositories: <span class="profile">${res.data.public_repos}</span>`;
-        urlContainer.innerHTML = `URL: <a href="${res.data.html_url}"><span class="profile">${res.data.html_url}</span></a>`;
-        followerContainer.innerHTML = `Followers: <span class="profile">${res.data.followers}</span>`;
-        followingContainer.innerHTML = `Following: <span class="profile">${res.data.following}</span>`;
-    })
+        const { name,login,public_repos,profile,followers,following,html_url } = res.data;
+        nameContainer.innerHTML = `Name: <span class="profile">${name}</span>`;
+        userContainer.innerHTML = `Username: <span class="profile">${login}</span>`;
+        reposContainer.innerHTML = `Repositories: <span class="profile">${public_repos}</span>`;
+        urlContainer.innerHTML = `URL: <a href="${res.data.html_url}"><span class="profile">${html_url}</span></a>`;
```
Comment (endormi) [inline js/app.js]: If you could fix the typo ${res.data.html_url} and just use ${html_url)

File: js/app.js [hunk 2]
```diff
@@ -24,13 +24,14 @@ const fetchUser = async (user) => {
...
 const showData = () => {
     fetchUser(inputValue.value).then((res) => {
-        nameContainer.innerHTML = `Name: <span class="profile">${res.data.name}</span>`;
-        userContainer.innerHTML = `Username: <span class="profile">${res.data.login}</span>`;
-        reposContainer.innerHTML = `Repositories: <span class="profile">${res.data.public_repos}</span>`;
-        urlContainer.innerHTML = `URL: <a href="${res.data.html_url}"><span class="profile">${res.data.html_url}</span></a>`;
-        followerContainer.innerHTML = `Followers: <span class="profile">${res.data.followers}</span>`;
-        followingContainer.innerHTML = `Following: <span class="profile">${res.data.following}</span>`;
-    })
+        const { name,login,public_repos,profile,followers,following,html_url } = res.data;
```
Comment (endormi) [inline js/app.js]: If you could use spaces in the consts, I just prefer it that way

**Comment by endormi** [comment]
Thanks for this Pull Request @MohammadHasham !

File: js/app.js (hunk 2 above)
Comment (MohammadHasham) [reply js/app.js]: Sure, I'll do that.

File: js/app.js (hunk 1 above)
Comment (MohammadHasham) [reply js/app.js]: Fixed.

File: js/app.js (hunk 2 above)
Comment (MohammadHasham) [reply js/app.js]: Fixed

**Comment by MohammadHasham** [comment]
Fixed! The changes have been made.

**Comment by endormi** [comment]
I accidentally created a revert branch, if you can (I don't have the authority) delete the branch :smile:

**Comment by MohammadHasham** [comment]
Done.

**Comment by endormi** [comment]
Thanks!

**Pull request merged**